plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -2641,3 +2641,118 @@ func TestPodToleratesTaints(t *testing.T) {
 		}
 	}
 }
+
+func makeNonBestEffortResources(milliCPU int64, memory int64) api.ResourceList {
```

Comment:
nit: just call this `getResourceList` ?

see an example: https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/resourcequota/resource_quota_controller_test.go#L33

There is nothing here that actually guaranteed its not best effort since you can pass 0 for both values.